package.json

```diff
@@ -17,9 +18,12 @@
     "release": "standard-version"
   },
   "dependencies": {
+    "@nanostores/preact": "^0.1.3",
```

Comment:
Can you run `ncu` and update all dependencies? I think now's a good time to do that.